app/components/Modal.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Modal as BootstrapModal, ModalBody, ModalFooter } from 'reactstrap'
+
+export default class Modal extends React.Component {
+  constructor (props) {
+    super(props)
+    this.getFooter = this.getFooter.bind(this)
+  }
+
+  getFooter () {
+    const { t } = this.props
+    switch (this.props.type) {
```

Comment:
Why are you choosing a `switch` here since there's only one option? Unless you are planning to have more options :thinking: